extensions/devservices/keycloak/src/main/java/io/quarkus/devservices/keycloak/KeycloakAdminPageBuildItem.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.devservices.keycloak;
+
+import java.util.Comparator;
+import java.util.List;
+
+import io.quarkus.builder.item.MultiBuildItem;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+
+public final class KeycloakAdminPageBuildItem extends MultiBuildItem {
+
+    public static final int DEFAULT_PRIORITY = 0;
+
+    private final CardPageBuildItem cardPageBuildItem;
+    /**
+     * Determines which {@link #cardPageBuildItem} is selected.
+     * Higher value means higher priority.
+     */
+    private final int priority;
```

Comment:
What is the idea behind the `priority` ?

Reply:
Yeah, I though we will discuss that :-D.

So far, we had one card created for Quarkus OIDC extension when Keycloak Dev Services were started. This card contained "Keycloak Provider" and "Keycloak Admin" links.

When we added support for Keycloak Dev Service started when Quarkus OIDC Client extension is used without Quarkus OIDC extension, card with "Keycloak Admin" was created for Quarkus OIDC Client. That is there are not 2 cards, always one card with "Keycloak Admin" link.

And that is what priority is for, to have just one card with "Keycloak Admin" link. I think it would be useless when for example all three OIDC, OIDC Client and OIDC Client registration were present, we had 3 cards with same link. And then imagine someone will also use Keycloak Admin Client in the future, will we show 4 cards with same link?

So, that is what priority is for, have status quo when Quarkus OIDC is present, everything will appear as it used to be one or 2 releases before (don't remember when that PR got merged).

Reply:
BTW I have tested it.

Reply:
Michal, I'm not sure we need this optimization, while it would indeed look strange to have `4/5` cards loaded, each of them having a `Keycloak Admin` link, in reality, we'll never have `4/5` cards loaded, `3` would be absolute max, `2` would be the main number IMHO, when more than 2 OIDC related extensions combine.

When I have for example `quarkus-oidc-client` and `quarkus-oidc-extension` or `quarkus-oidc-client-registration` and `quarkus-oidc-extension` loaded, I definitely would like to see 2 cards, one for `quarkus-oidc`, and another one for either  `quarkus-oidc-client` or `quarkus-oidc-client-registration`...

I don't think having a single card to represent `quarkus-oidc`, `quarkus-oidc-client`, etc when more than one OIDC or Keycloak aware extensions are loaded  would be nice...

May be we can simply agree that if `quarkus-oidc` is loaded then `Keycloak Admin` is shown only in the `quarkus-oidc` card, if `quarkus-oidc` is not loaded - then it is shown in each of the other OIDC aware cards

Reply:
When I don't have strong opinion, I just keep original behavior because it is something we are most likely to agree on. I don't really mind here, I added this priority because I thought you will want to keep how it worked. Let me drop priority altogether. Then you can test it and if it will look good to you, I am fine as well.

Reply:
Done. I have upgraded my Fedora OS version today and had to drop Docker for the upgrade to work. So I didn't test this last change yet, will check tomorrow when I convince Podman to work, but I expect the last commit to work just fine.

Reply:
I have Podman working so I checked changes. IMO it looks ok:
![image](https://github.com/user-attachments/assets/5b0d901e-eaef-452b-be6d-9c9036bc590a)

Reply:
Thanks @michalvavrik